src/app/components/select-field/index.html

```diff
@@ -164,33 +164,34 @@
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="headingText"
       isOptional="true"
     >
-      The text provided to the select field's <sky-code>headingText</sky-code> property. If your custom picker is a modal, use this property for the modal's title.
+      Specifies a header for the select field component's modal and provides it to the select field's <sky-code>pickerHeading</sky-code> property. This property accepts <sky-code>string</sky-code> values.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="selectedValue"
+      propertyName="selectMode"
       isOptional="true"
     >
-      The select field's current value. If the select field's <sky-code>selectMode</sky-code> is set to <sky-code>single</sky-code>, this will be a <sky-code>SkySelectField</sky-code> object; otherwise it will be an array of <sky-code>SkySelectField</sky-code> objects. Use this property to select the corresponding items in the custom picker.
+      Specifies the selection mode that determines whether users can select one item or multiple items and provides the value to the select field's <sky-code>selectMode</sky-code> property.
     </sky-demo-page-property>
     <sky-demo-page-property
-      propertyName="selectMode"
+      propertyName="selectedValue"
       isOptional="true"
     >
-      The value provided to the select field's <sky-code>selectMode</sky-code> property. Use this property to determine whether the custom picker should allow more than one value to be selected.
+      Specifies the select field's current value. If <sky-code>selectMode</sky-code> is set to <sky-code>single</sky-code>, then this value is <a stacheRouterLink="." fragment="skyselectfield-properties">a <sky-code>SkySelectField</sky-code> object</a>. If <sky-code>selectMode</sky-code> is set to <sky-code>multiple</sky-code>, then this value is an array <a stacheRouterLink="." fragment="skyselectfield-properties">of <sky-code>SkySelectField</sky-code> objects</a>. Use this property to select the corresponding items in the custom picker.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="showAddNewRecordButton"
       isOptional="true"
+      defaultValue="false"
     >
-      The value provided to the select field's <sky-code>showAddNewRecordButton</sky-code> property. Use this property to determine whether a new record can be added from the custom picker when the desired value is not in the list of options.
+      Indicates whether to display a button for users to add new items directly from the select field modal and provides the value to the select field's <sky-code>showAddNewRecordButton</sky-code> property. This property accepts <sky-code>boolean</sky-code> values.
```

Comment:
The rewording to "provides the value to the select field's `showAddNewRecordButton`" is problematic here for the same reasons as the ones stated above.